### DISCUSSION TITLE
Error in command readseq Chapter VII DATA FORMATS-Introduction to data formats

### DISCUSSION THREAD

**Discussion by rqportillo** [biostars/biostar-handbook-issues#235] [Q&A]
I found a mistake in a flag in the command readseq. The error was encountered in the Chapter VII DATA FORMATS-Introduction to data formats.

The command in the book is:
cat NC_001501.gb | readseq -p -format FASTA > NC_001501-version3.fa

The correct format will be "-f" or "-format=" instead of -format only.

Running the command with both corrections:

Option 1:
cat NC_001501.gb | readseq -p -f FASTA > NC_001501-version3.fa

Option 2:
cat NC_001501.gb | readseq -p -format=FASTA > NC_001501-version3.fa
 
Creates the file correctly.

**Comment by ialbert** [comment]
Could you please link to the page that has this information.

 I have a hard time locating this line.

**Comment by rqportillo** [comment]
Hi Istvan,

The page is:

https://www.biostarhandbook.com/introduction-to-data-formats.html

Raul.

**Reply by ialbert** [reply]
Where is this page linked from? That is the root of the problem. 

Technically this page should not exist anymore because it is not part of the current edition. It is a remnant of an older version of the book. If you load the main book page from, you can't find this page listed in the table of contents or the search. The content has been moved to:

https://www.biostarhandbook.com/reformatting-data.html

Looks like somewhere, an older link remained. I have not deleted old pages, so outdated links at least load some content. It would be worse if you get a 404, but I should go through and perhaps mark these pages as deprecated to help the readers.
